packages/styles/stepper.css

```diff
@@ -0,0 +1,87 @@
+.Stepper {
+  --step-indicator-size: 25px;
+  --step-line-height: 8px;
+  --step-line-offset: 18px;
```

Comment:
No vars? Wouldn't we at least want to make these relative to the font size?

Reply:
Made the change to `em` -- good call 👍